sidebars.ts

```diff
@@ -16,7 +16,17 @@ const sidebars: SidebarsConfig = {
         {
             type: "category",
             label: "Commands",
-            items: [{ type: "autogenerated", dirName: "commands" }],
+            items: [
+                { type: "category", label: "browser", items: [{ type: "autogenerated", dirName: "commands/browser" }] },
+                { type: "category", label: "element", items: [{ type: "autogenerated", dirName: "commands/element" }] },
+                {
+                    type: "category",
+                    label: "expect",
+                    items: [{ type: "autogenerated", dirName: "commands/expect" }],
+                    link: { type: "doc", id: "commands/expect/overview" },
+                },
+                { type: "category", label: "mock", items: [{ type: "autogenerated", dirName: "commands/mock" }] },
+            ],
```

Comment:
I dont like being forced to describe each chapter manually, but i haven't find a better alternative

Reply:
it's the only way